.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
```

Comment:
This is Prettier's default value so I'm not sure if it makes sense to override it.